client/express-checkout/utils/test/normalize.js

```diff
@@ -263,6 +264,66 @@ describe( 'Express checkout normalization', () => {
 		} );
 	} );
 
+	describe( 'normalizePayForOrderData', () => {
+		test( 'should normalize pay for order data with complete event and paymentMethodId', () => {
+			window.wcpayFraudPreventionToken = 'token123';
+
+			const event = {
```

Comment:
Nit: Could this be simplified to the following?
```javascript
const event = {
    billingDetails: {},
    shippingAddress: {},
    shippingRate: {},
    expressPaymentType: 'express',
};
```

Reply:
It could, but I intentionally left it like that because that's how the event looks like. If we work on passing all that info to the backend so that the order is filled in with it, we'll need to tweak the `expect`, but the event is up-to-date.